ext/pcre/php_pcre.stub.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/** @return int|false */
+function preg_match(string $pattern, string $subject, &$subpatterns = null, int $flags = 0, int $offset = 0) {}
+
+/** @return int|false */
+function preg_match_all(string $pattern, string $subject, &$subpatterns = null, int $flags = 0, int $offset = 0) {}
+
+/**
+ * @param string|array $regex
+ * @param string|array $replace
+ * @param string|array $subject
+ * @return string|array|null
+ */
+function preg_replace($regex, $replace, $subject, int $limit = 0, &$count = null) {}
+
+/**
+ * @param string|array $regex
+ * @param string|array $replace
+ * @param string|array $subject
+ * @return string|array|null
+ */
+function preg_filter($regex, $replace, $subject, int $limit = 0, &$count = null) {}
+
+/**
+ * @param string|array $regex
+ * @param string|array $subject
+ * @return string|array|null
+ *
+ * TODO: $callback should be `callable`
+ */
+function preg_replace_callback($regex, $callback, $subject, int $limit = 0, &$count = null) {}
+
+/**
+ * @param array $pattern
+ * @param string|array $subject
+ * @param int $limit
+ * @param int $count
```

Comment:
To be consistent with the others, I think we should drop the `@param`s that have type info in the signature.